includes/class-sensei-course.php

```diff
@@ -3028,7 +3033,11 @@ public static function can_current_user_manually_enrol( $course_id ) {
 	public static function output_course_enrolment_actions() {
 		global $post;
 
-		$is_course_content_restricted = (bool) apply_filters( 'sensei_is_course_content_restricted', false, $post->ID );
+		$is_course_content_restricted = (bool) apply_filters_deprecated(
```

Comment:
Should anything be done with the usage of this variable below at this point?

Reply:
Yes! After removing this filter we should remove the `$is_course_content_restricted` from the `if`.

`if ( $is_course_content_restricted && false == $should_display_start_course_form ) {`

Do you think we should add a comment on the code or something like that?

Reply:
Maybe we could add a comment in the issue where we are tracking the removal for this? Just to remind us that there are some extra things that need to be removed here.

Reply:
Sure! I'll do that! =)

Reply:
Added here: https://github.com/Automattic/sensei/issues/2976 ;)